test/common/testlib.py

```diff
@@ -1074,6 +1250,33 @@ def arg_parser():
     parser.set_defaults(verbosity=1, fetch=True)
     return parser
 
+
+def browser_path(headless=True):
+    """Return path to CDP browser.
+
+    Support the following locations:
+     - "chromium-browser" in $PATH (distro package)
+     - /usr/lib*/chromium-browser/headless_shell (chromium-headless RPM), if
+       headless is true
+     - node_modules/chromium/lib/chromium/chrome-linux/chrome (npm install chromium)
```

Comment:
Is it necessary to build-depend on chromium? When running locally, I'm fine with just having the exception thrown when `which` fails, but we did properly depend on phantom-js.

Reply:
What do you mean with "build-depend"? This only adds `chrome-remote-interface` to `devDependencies`, which is tiny. `make check` will just skip the TAP tests if chromium/phantom are not installed (that's how we run unit tests on the build in our VM images). Or do you mean we should *introduce* a build-dependency on chromium? IMHO that would be overkill.

Reply:
Indeed, phantom is an `optionalDependency`. You're right, let's not introduce a huge dependency if we didn't have it before. Sorry about the noise.